boolean/test_boolean.py

```diff
@@ -180,9 +249,9 @@ def test_eval(self):
         a = boolean.Symbol("a")
         self.assertTrue(~a == ~a)
         self.assertFalse(a == boolean.parse("~~a", eval=False))
-        self.assertTrue(a == ~~a)
-        self.assertTrue(~a, ~~~a)
-        self.assertTrue(a, ~~~~a)
+        self.assertTrue(a == ~ ~a)
+        self.assertTrue(~a, ~ ~ ~a)
```

Comment:
Actually even in the original, `self.assertTrue(~a, ~~~a)` is a buggy test ...
the second arg `~~~a` is the message in case of assertion failure .....
So the tests is not testing anything. It will need to be fixed.